.changeset/hot-schools-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sumup/circuit-ui': major
```

Comment:
Since this is a major change for Circuit UI v6, we'll point this PR to a new "next" branch, so we don't block the main release channel while we're working on v6.

More docs on the process here: https://circuit.sumup.com/?path=/docs/introduction-contributing-release-process--page

You don't have to do anything, I'll make the new branch soon and update this PR to point to it instead 🙂

Reply:
Sounds good! Thanks @robinmetral !

Reply:
Base changed to `next` ✅